nightcord.js.org

RellyLegend opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your domain request, you need to create a file named `CNAME` in the root of your github pages site (_this may not be the root of your repository_). 📝
This file should contain a single line with the full js.org domain you are requesting, in this instance `nightcord.js.org`.

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But as Matt said your CNAME is not in the correct place. You have to move it to the "docs" folder. FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
I'll merge after we have "contact". Please keep this PR open.

RellyLegend commented:
Hi, I'm not sure If I am suppose to reply but I'd like to inform that I
have fixed the issue and added CNAME to my github pages root directory.

On Thu, Sep 2, 2021 at 2:42 PM Stefan Keim ***@***.***> wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. But as
> Matt said your CNAME is not in the correct place. You have to move it to
> the "docs" folder. FYI:
> https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
> I'll merge after we have "contact". Please keep this PR open.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/js.org/pull/6328#issuecomment-911697821>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASCCY5FRPVBVFWCDMBI3E2LT755LRANCNFSM5DIX4OBQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

indus commented:
Welcome https://nightcord.js.org
